bloom-housing/bloom
# 2919/progress bar update

# Pull Request Template

## Issue Overview

This PR addresses #2919 


## Description

This PR updates the Progress Nav component to utilize v2 styling, include more accessible labeling, and add a unique styling option (referred to as bar style). 

**Also, this PR includes a few updates to the pre-existing dot styling to include the same default font sizing, font coloring, and weight as the [Figma](https://www.figma.com/file/pIlN9vqLwUgOZYq6I5oM9l/bloom-public-eligibility-wizard?node-id=827%3A7153) (per discussions with Jesse).**

## How Can This Be Tested/Reviewed?

This can be tested by looking through the Progress Nav component on storybook. Additionally, by signing in on the Public side and applying to a listing, you'll see that the existing progress nav component styling is still functioning as you fill out the application.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
